experiments/static/experiments/js/experiments.js

```diff
@@ -4,7 +4,7 @@ experiments = function() {
             $.post("/experiments/confirm_human/");
         },
         goal: function(goal_name) {
-            $.post("/experiments/goal/" + goal_name);
+            $.post("/experiments/goal/" + goal_name + "/");
```

Comment:
I think it's cleaner to use f-Strings
```suggestion
            $.post(f"/experiments/goal/{goal_name}/");
```

Reply:
@iamjazzar This is JavaScript 😀. Good try, I attempt, I almost released this.